pkg/preparer/orchestrate.go

```diff
@@ -189,7 +190,18 @@ func (p *Preparer) handlePods(podChan <-chan ManifestPair, quit <-chan struct{})
 				if pod.Id == POD_ID {
 					pod.DefaultTimeout = time.Duration(0)
 				}
-				pod.SetLogBridgeExec(p.logExec)
+
+				effectiveLogBridgeExec := p.logExec
+				// pods that are in the blacklist for this preparer shall not use the
+				// preparer's log exec. Instead, they will use the default svlogd logexec.
+				for _, podID := range p.logBridgeBlacklist {
+					if string(pod.Id) == podID {
```

Comment:
nit: `pod.Id.String()`. This makes it easier for the compiler to validate things if we ever change the underlying type although I admit that's unlikely